wfl/fit/gap/multistage.py

```diff
@@ -82,6 +83,28 @@ def _select_info(ats, info_keys):
                 del at.info[k]
 
 
+def get_Num_bonds(slab):
```

Comment:
Why is this function in `gap/multistage.py`? Also, it'd almost certainly be much faster to use the linear scaling ASE neighborlist https://wiki.fysik.dtu.dk/ase/ase/neighborlist.html#ase.neighborlist.neighbor_list, or even better the matscipy one which has identical semantics (although different spelling of the functions)

Reply:
This part has been removed from multistage.py file